packages/spacecat-shared-rum-api-client/test/fixtures/high-organic-low-ctr.json

```diff
@@ -4,26 +4,47 @@
     "page": "https://www.spacecat.com/about-us",
     "screenshot": "",
     "trackedPageKPIName": "Click Through Rate",
-    "trackedPageKPIValue": 0.14099783080260303,
+    "trackedPageKPIValue": 0.14316702819956617,
     "pageViews": 46100,
     "samples": 46100,
+    "siteAverage": 0.40828402366863903,
     "metrics": [
```

Comment:
types of the items in the `metrics` array are `object`, `object`, `array`, `array`, ... is this intended?

Reply:
no, that's not intended, just saw that yesterday, fixing it.